sec-scanners-config.yaml

```diff
@@ -2,6 +2,7 @@ module-name: cloud-manager
 rc-tag: 1.1.0
 protecode:
   - europe-docker.pkg.dev/kyma-project/prod/cloud-manager:1.1.0
+  - europe-docker.pkg.dev/kyma-project/prod/cloud-manager:main
```

Comment:
The image above the one you added is obsolete, we are on 1.1.1.
Please consider updating that as well.

Also, should rc-tag be changed as well? To `main` perhaps?

Reply:
I'm not sure whether we should change it or add it.

Reply:
As others do, on main it should be main, on release tag it should be release tag